src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,28 @@
 import React from 'react';
+import './CommentList.scss';
+import { Comment } from '../../types/Comment';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  comments: Comment[] | null;
```

Comment:
comments prop should not accept null. Use an empty array for default value instead.